e2e/playwright/file-tree.spec.ts

```diff
@@ -238,6 +238,26 @@ test.describe('when using the file tree to', () => {
     }
   )
 
+  test(
+    `create new folders and that doesn't trigger a navigation`,
+    { tag: '@electron' },
```

Comment:
Will this test cover platform-specific code? If so, we may want to opt in `@macos` and `@windows`. If not, feel free to ignore.

Reply:
Great point, fixing now